metals/src/main/scala/scala/meta/internal/bsp/ConnectionBspStatus.scala

```diff
@@ -55,4 +114,80 @@ object ConnectionBspStatus {
       command = ServerCommands.ConnectBuildServer.id,
       commandTooltip = "Reconnect.",
     ).withStatusType(StatusType.bsp)
+
+  def bspErrorParams(
+      serverName: String,
+      icons: Icons,
+      message: String,
+      errorsNumber: Int,
+  ): MetalsStatusParams =
+    MetalsStatusParams(
+      s"$serverName $errorsNumber ${icons.alert}",
+      "warn",
+      show = true,
+      tooltip = message.trimTo(TOOLTIP_MAX_LENGTH),
+      command = ServerCommands.RunDoctor.id,
```

Comment:
As a follow up, would it be possible (probably on vs-code side) to make doctor open on the part with error reports?

Reply:
Yeah, that sounds like a good idea.

Reply:
We should also open them in that case, currently they are folded.